src/System.Private.ServiceModel/tests/Common/Infrastructure/src/testproperties.props

```diff
@@ -105,7 +105,11 @@
     <PropertyGroup Condition="'$(Kerberos_Available)' == ''">
       <Kerberos_Available>false</Kerberos_Available>
     </PropertyGroup>
-  
+
+    <PropertyGroup Condition="'$(ServiceUri)' == ''">
+      <ServiceUri >localhost</ServiceUri >
```

Comment:
1. Is this necessarily correct to have the ServiceUri be this by default? 
2. Should we split out the hostname and basepath like we talked about earlier?